src/plugins/index.js

```diff
@@ -36,6 +37,42 @@ export default class Plugins {
     this.lock = new Lock(this.out)
   }
 
+  clearCache (...paths: string[]) {
+    this.cache.deletePlugin(...paths)
+  }
+
+  async _installCheck (name: string, tag: string) {
```

Comment:
@dickeyxxx the [Plugins](https://github.com/heroku/cli-engine/pull/211/files#diff-910474ccfd83e0d91c039c059135004fR75) class now inherits from this.  Basically the migrator should not be loading plugins so I split the class so that migrator used only a subset of the plugin methods to avoid accidentally loading.